make/common/templates/ui/env

```diff
@@ -6,3 +6,4 @@ GODEBUG=netdns=cgo
 ADMINSERVER_URL=$adminserver_url
 UAA_CA_ROOT=/etc/ui/certificates/uaa_ca.pem
 _REDIS_URL=$redis_url
+REGISTRY_CONTROLLER_URL=$registry_controller_url
```

Comment:
Does UI need to talk to the controller directly? If it does, put it into the env of adminserver and UI gets the URL from adminserver.